canvas_modules/harness/src/client/App.js

```diff
@@ -2674,6 +2676,7 @@ class App extends React.Component {
 			heading: this.state.heading,
 			useHeading: this.useHeading,
 			light: this.state.light,
+			trimSpaces: this.state.trimSpaces,
```

Comment:
This is not doing anything. Can you follow all the way through and have this exposed in the test harness side panel so it can easily be tested?

Also need to update the test harness visual docs to include the new option